apps/api/v2/src/modules/oauth/oauth-client.controller.ts

```diff
@@ -47,8 +45,7 @@ export class OAuthClientController {
 
   @Get("/")
   @HttpCode(HttpStatus.OK)
-  async getOAuthClients(@Res({ passthrough: true }) res: Response) {
-    const userId = res.locals.apiKey?.userId;
+  async getOAuthClients(@GetUser("id") userId: number) {
     return this.oauthClientRepository.getUserOAuthClients(userId);
```

Comment:
@exception where can we define in nest.js a generic response handler ? making sure that we always return some thing to the user in the same format ?

like 

```ts
{
status: "OK",
data: []
}
```

{
status: "error",
error: ...
}
```